README.md

```diff
@@ -103,6 +135,222 @@ curl -XPOST -v \
+* ```vetted```: **(required)** the email sent when the Registrant has successfully vetted a Second Factor.
+
+The following list of emails is intended to be used in the future, 
+the functionality requiring these is not yet implemented. 
+* ```registration_code_expiration_warning```: the email sent when the Registrant has not vetted his Second Factor after 1 week.
+* ```second_factor_expiration_first_reminder```: the email sent when the Second Factor has not been used for 5 months
+* ```second_factor_expiration_second_reminder```: the email sent when the Second Factor has not been used for 5 months + 2 weeks.
+* ```second_factor_revocation_confirmation```: the email sent when a Second Factor has successfully been revoked.
+
+Each email contains an object, where each property corresponds with an IETF language tag (2 letter lower cased language code + underscore + 2 letter upper cased country code, i.e. nl_NL, nl_BE) that may be supported in the application.
+
+```json
+"email_templates": {
+    "confirm_email": {
+        "nl_NL": "Volledige template met een {{ variableName }} variabele in Twig syntax. May include <b>HTML</b> and 
+new lines.",
+        "en_GB": "Full template with a {{ variableName }} variable in Twig syntax"
+    }
+}
+```
+
+### Processing
+There will only be validation if the required email-template properties are present, 
+each with at least the default language ("en_GB") template available. 
+All previous templates will be removed from the database and the new templates will be inserted. 
+
+### Template Variables
+
+##### e-mail verification (confirm_email)
+| variable name   | type   | example                                                 |
+|-----------------|--------|---------------------------------------------------------|
+| commonName      | string | Jan Modaal                                              |
+| email           | string | jan@modaal.nl                                           |
+| verificationUrl | string | http://self-service.com/verify-email?n=0123456789abcdef |
+
+#### registration (registration_code)
+<table>
+	<thead>
+		<tr>
+			<th>
+			<p>variable name</p>
+			</th>
+			<th>
+			<p>type</p>
+			</th>
+			<th>
+			<p>example</p>
+			</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td>commonName</td>
+			<td>string</td>
+			<td>Jan Modaal</td>
+		</tr>
+		<tr>
+			<td>email</td>
+			<td>string</td>
+			<td>jan@modaal.nl</td>
+		</tr>
+		<tr>
+			<td>registrationCode</td>
+			<td>string</td>
+			<td>ABC23456</td>
+		</tr>
+		<tr>
+			<td colspan="1">ras</td>
+			<td colspan="1">array</td>
+			<td colspan="1">
+			<table border="0" cellpadding="0" cellspacing="0">
+				<tbody>
+					<tr>
+						<td>
+						<p><code>[</code></p>
+						<p><code>&nbsp;&nbsp;&nbsp;&nbsp;</code><code>[</code></p>
+						<p><code>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</code><code>&#39;commonName&#39;</code>&nbsp;<code>=&gt;&nbsp;</code><code>&#39;Jan Modaal&#39;</code><code>,</code></p>
```

Comment:
Nit: There is a comma in this array, yet there are no comma's after the other array entries.